content/applications/websites/livechat/responses.rst

```diff
@@ -1,48 +1,176 @@
+ticket.
+
+After entering the `/helpdesk` command, select a title for the ticket and type it into the chat
+window, then press `Enter`.
+
+.. image:: responses/helpdesk.png
    :align: center
-   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat
+   :alt: View of the results from a helpdesk search in a Live Chat conversation.
 
-.. tip::
-   Helpdesk tickets created from the chat automatically add the conversation as a description of
-   the ticket. The same goes for the creation of a lead.
+The newly created ticket will be added to the *Helpdesk* team that has live chat enabled. If more
+than one team has live chat enabled, the ticket will automatically be assigned based on the team's
+priority.
+
+The transcript from the conversation will be added to the new ticket, under the
+:guilabel:`Description` tab.
+
+To access the new ticket, click on the link in the chat window, or go to the
+:menuselection:`Helpdesk app` and click the :guilabel:`Tickets` button on the kanban card for the
+appropriate team.
+
+Search for a ticket from a live chat
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If an operator types `/helpdesk_search` in the chat window, they can search through *Helpdesk*
+tickets by ticket number or keyword.
+
+After entering the `/helpdesk_search` command, type a keyword or ticket number, then press `Enter`.
+If a related ticket is found, a link will be generated in the conversation window.
+
+.. image:: responses/helpdesk-search.png
+   :align: center
+   :alt: View of the results from a helpdesk search in a Live Chat conversation.
+
+History
+-------
+
+If an operator types `/history` in the chat window, it will generate a list of the most recent pages
+the visitor has viewed on the website (up to 15).
 
-Send canned responses
-=====================
+.. image:: responses/responses-history.png
+   :align: center
+   :alt: View of the results from a /history command in a Live Chat conversation.
+
+Lead
+----
 
-| Canned responses allow you to have a full piece of text being placed when you type a shortcut word.
-  To create them, go to :menuselection:`Live Chat --> Configuration --> Canned Responses`.
-| To use them during a conversation, simply type **:** followed by the shortcut word you created.
+By typing `/lead` in the chat window, an operator can create a *lead* in the *CRM* application.
+After typing `/lead`, create a title for this new lead, then press `Enter`.
 
-.. image:: responses/canned_response.png
+.. image:: responses/responses-lead.png
    :align: center
-   :alt: View of a chat window and the use of a canned response in Odoo Live Chat
+   :alt: View of the results from a /lead command in a Live Chat conversation.
+
+.. important::
+   The `/lead` command can only be used if the *CRM* app has been installed.
+
+To access the lead, click the link in the chat window, or navigate to the :menuselection:`CRM` app
```

Comment:
What link?
Is there one that appears after typing `/lead`, typing a title, and pressing enter?
Is that link viewable by only operators or can the customer see it too?